src/Plugin/Derivative/TranslationLocalTasks.php

```diff
@@ -0,0 +1,82 @@
+<?php
+
+namespace Drupal\oe_translation\Plugin\Derivative;
+
+use Drupal\Component\Plugin\Derivative\DeriverBase;
+use Drupal\Core\Plugin\Discovery\ContainerDeriverInterface;
+use Drupal\Core\StringTranslation\StringTranslationTrait;
+use Drupal\oe_translation\TranslatorProvidersInterface;
+use Symfony\Component\DependencyInjection\ContainerInterface;
+
+/**
+ * Generates the local tasks for our translation system.
+ */
+class TranslationLocalTasks extends DeriverBase implements ContainerDeriverInterface {
+
+  use StringTranslationTrait;
+
+  /**
+   * The base plugin ID.
+   *
+   * @var string
+   */
+  protected $basePluginId;
+
+  /**
+   * The translation providers service.
+   *
+   * @var \Drupal\oe_translation\TranslatorProvidersInterface
+   */
+  protected $translatorProviders;
+
+  /**
+   * Creates a FieldUiLocalTask object.
+   *
+   * @param string $base_plugin_id
+   *   The base plugin ID.
+   * @param \Drupal\oe_translation\TranslatorProvidersInterface $translatorProviders
+   *   The translation providers service.
+   */
+  public function __construct($base_plugin_id, TranslatorProvidersInterface $translatorProviders) {
+    $this->basePluginId = $base_plugin_id;
+    $this->translatorProviders = $translatorProviders;
+  }
+
+  /**
+   * {@inheritdoc}
+   */
+  public static function create(ContainerInterface $container, $base_plugin_id) {
+    return new static(
+      $base_plugin_id,
+      $container->get('oe_translation.translator_providers'),
+    );
+  }
+
+  /**
+   * {@inheritdoc}
+   */
+  public function getDerivativeDefinitions($base_plugin_definition) {
+    $this->derivatives = [];
+
+    $entity_types = $this->translatorProviders->getDefinitions();
+    foreach ($entity_types as $entity_type_id => $entity_type) {
+      $this->derivatives["$entity_type_id.dashboard"] = [
+        'route_name' => "entity.$entity_type_id.content_translation_overview",
+        'title' => $this->t('Dashboard'),
+        'parent_id' => "content_translation.local_tasks:entity.$entity_type_id.content_translation_overview",
+        'weight' => 1,
+      ] + $base_plugin_definition;
+
+      // @todo change this to the proper route for local translations.
+      $this->derivatives["$entity_type_id.local_translations"] = [
+        'route_name' => "entity.$entity_type_id.canonical",
```

Comment:
Looks a bit weird to have it in the canonical, I would move it to `entity.$entity_type_id.content_translation_overview` to have it on the same page with the Dashboard but I guess it's fine here too until we have the route..

Reply:
Yeah, it's fine like this.